en/editor/scene/index.md

```diff
@@ -23,9 +23,17 @@ In 2D view, you can move and rotate view of **Scene Editor** by following operat
 - **F** shortcut：focus editor to the selected node.
 
 
-### Coordinate system and grid
+### Coordinate System And Grid
 Grid in the scene is important reference information for us to place the scene elements. For information of the relationship between coordinate system and node attributes like position, please read [Transform](../../concepts/scene/coord.md) section.
```

Comment:
Grid in the scene is important reference information for us to place the scene elements. For information of the relationship between coordinate system and node attributes like position, please read [Transform](../../concepts/scene/coord.md) section.
-->
The __Grid__ in a scene is an important reference to layout scene elements. For information on the relationship between coordinate systems and node attributes, like position, please read the [Transform](../../concepts/scene/coord.md) documentation.